### PR TITLE
Dev/header

### PR DESCRIPTION
**Pull request by arjund-06**
### Functionality:
Makes Navbar more funtional.

### Solution:
Grouped the create task and start sprint buttons and grouped the board buttons and added button to create team and add members.

### Risk level:
- [ ] high 
- [ ] medium
- [X] low

### How to test:
1. Use ng serve and see the difference in the Navbar.

### CODE REVIEW

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
make a separate pr for header and patch

File: src/app/app.module.ts [hunk 1]
```diff
@@ -63,11 +64,14 @@ import { RangeCardComponent } from "./body/my-dashboard/performance-chart/range-
...
     StatusIconsComponent,
     FilterTaskComponent,
+    FilterTaskComponent,
     NavbarComponent,
     ChartComponent,
     MyTasksComponent,
     PerformanceChartComponent,
-    RangeCardComponent
+    RangeCardComponent,
+    FilterTaskComponent,
```
Comment (shivamgoyal7) [inline src/app/app.module.ts:73]: remove FilterTaskcomponent it is added three times.Check it

File: src/app/app.module.ts [hunk 2]
```diff
@@ -38,7 +38,8 @@ import { NavbarComponent } from './body/navbar/navbar.component';
 import { ChartComponent } from './body/common-tools/chart/chart.component';
 import { MyTasksComponent } from './body/my-dashboard/my-tasks/my-tasks.component';
 import { PerformanceChartComponent } from './body/my-dashboard/performance-chart/performance-chart.component';
-import { RangeCardComponent } from "./body/my-dashboard/performance-chart/range-card/range-card.component"
+import { RangeCardComponent } from "./body/my-dashboard/performance-chart/range-card/range-card.component";
```
Comment (shivamgoyal7) [inline src/app/app.module.ts:41]: add semicolon

File: src/app/body/task-details/task-details.component.html [hunk 3]
```diff
@@ -83,6 +87,12 @@ <h5 class="active nav-link navbar-text" style="font-weight:bolder">Task Details<
                                 <div id="storyPointNumberTaskDescription">{{task.StoryPointNumber}}</div>
                             </div>
                         </div>
+                        <div class="form-group col-md-4">
+                            <label for="inputState">Work Done</label>
+                            <div class="alert alert-info form-inline" role="alert">
+                                <div id="workDoneTaskDescription">{{task.WorkDone}}</div>
+                            </div>
+                        </div>
```
Comment (shivamgoyal7) [inline src/app/body/task-details/task-details.component.html:95]: revert the changes from this file

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
do these changes asap

**Comment by arjund-06** [comment]
Closed because it had commits from two different tasks

**Pull request closed without merging**